v3/plugins/joaompinto/vscode-asciidoctor/latest/meta.yaml

```diff
@@ -0,0 +1,13 @@
+publisher: joaompinto
+apiVersion: v2
+version: latest
+type: VS Code extension
+name: vscode-asciidoctor
+displayName: AsciiDoc support
+title: AsciiDoctor Plugin.
+description: This extension provides a live preview, syntax highlighting and snippets for the AsciiDoc format using Asciidoctor flavor.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
```

Comment:
Change on https://raw.githubusercontent.com/asciidoctor/asciidoctor-vscode/v2.7.6/images/icon.png as 2.7.6 is ised for latest

Reply:
Wrong icon type